app/api/balance/model.go

```diff
@@ -5,12 +5,15 @@ import (
 	"backend/app/common/consts"
 	"backend/app/common/models"
 	"time"
+
+	"gorm.io/gorm"
 )
 
 type Balance struct {
 	ID      int `gorm:"primaryKey;autoIncrement" json:"id" validate:"required"`
 	Balance int `gorm:"not null"`
 	models.Timestamps
```

Comment:
sudah ado models yg softdelete,
```suggestion
	models.TimestampsSoftDelete
```
![image](https://github.com/user-attachments/assets/dfde2f15-4e9f-4ddf-a1e8-e7c4fcfe0448)